index.md

```diff
@@ -369,3 +369,33 @@ during the workshop.
 
 {% include install_instructions/git.html %}
 {% include install_instructions/python.html %}
+
+<h3>Instructions for 'Introduction to Machine Learning with scikit-learn' </h3>
+For this workshop, you need to have Python 3 installed. We work with Jupyter notebooks. See the instructions for the Intro to Python workshop on how to install Jupyter with Anaconda.
+
```

Comment:
Is the installation of python 3 covered in the python installation instructions? Do we need to mention it here if it is?

'We work with Jupyter notebooks' seems a little terse, but I'm struggling to find a better phrasing.